WcaOnRails/app/mailers/two_factor_mailer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class TwoFactorMailer < ApplicationMailer
+  def send_otp_to_user(user)
+    @user = user
+    @code = user.current_otp
+    mail(
+      to: @user.email,
+      from: "software@worldcubeassociation.org",
```

Comment:
I forgot to comment about this, but basically I did not put `notifications@` because of the recurrent issues with the WCT receiving emails because the email clients are not honoring the `reply_to` field.
Since issues with that should be redirected to the WST I put our email here.